content/post/data-dies-in-darkness.md

```diff
@@ -0,0 +1,73 @@
+* Disagreement or misalignment about how business phenomena or KPIs ought to be measured
+* Unnoticed "data drift" as business-logic assumptions or data integrations get stale
+
+My personal motto for improving data quality, adapted from [The Washington Post](https://www.washingtonpost.com/lifestyle/style/the-washington-posts-new-slogan-turns-out-to-be-an-old-saying/2017/02/23/cb199cda-fa02-11e6-be05-1a3817ac21a5_story.html?utm_term=.3b9fcbc03a44), is that "Data Dies in Darkness" (which is an appealingly more dramatic phrasing of "sunlight is the best disinfectant"). The easiest way to promote data quality is to have as many eyes on as many pieces of data as possible -- this way potential issues are quickly identified and fixed rather than "festering" and slowly degrading trust in the entire system.
+
+
+Most organizations can dramatically improve their data quality by following a few simple practices:
+1. Put everything in the data warehouse, and let the analysts own the business logic for data transformations 
+2. Identify the directly-responsible-individual for every business KPI
+3. Automate all data-driven workflows
+4. Dashboard everything, and send emailed reports (including relevant executives) at a regular cadence
+5. Make heavy use of "data quality" alerts and monitoring
+
+I'll take each of these recommendations one-by-one:
+
+#### Put everything in the data warehouse, and let the analysts own the business logic for data transformations 
+All data important to the business should be in the warehouse. Period. This allows for an easy demarcation between what is "approved" as the single source of truth for the business and what is "potentially unreliable." Additionally, there are strong "network" effects to data -- the more data you have in the warehouse, the more value you can get from all of the other data that are available.
+
+That analysts have the ability to make changes to (and the responsibility for!) is key because they are the link between the folks operating the business and the reporting tools. The analyst's role as "data doula" responsible for translating actual business concepts into data models is key, and your infrastructure should support a fast-feedback loop where the analysts can quickly work on and update data transformation logic in close partnership with the team or person who is operationally responsible for the data. If it takes weeks to get a ticket for a data-transformation change through your engineering organization, your partners will have already abandoned your tool (and lost faith in you).
+
+Businesses (and especially startups!) can change rapidly, and the data and reporting tools need to be able to change just as fast if they are to remain relevant. Training analysts to build and maintain these data tools is crucial to building a data platform that is sufficiently flexible to support the shifting needs of the business.
+
+#### Identify the directly-responsible-individual for every business KPI
```

Comment:
when I first read this, I thought it meant the person responsible for implementing and maintaining the warehouse implementation of the metric.